sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosConflictsTest.java

```diff
@@ -0,0 +1,461 @@
+                }
+
+                //We will see some conflicts, as there is no sproc to resolve
+                assertThat(testPojos.size()).isEqualTo(containers.size() - 1);
+
+                //Testing queryConflicts(String query, CosmosQueryRequestOptions cosmosQueryRequestOptions)
+                testPojos.clear();
+                String query = String.format("SELECT * from c where c.id in (%s)",
+                    Strings.join(conflictIds.stream().map(s -> "'" + s + "'").collect(Collectors.toList())).with(","));
+                CosmosQueryRequestOptions options = new CosmosQueryRequestOptions();
+                options.setPartitionKey(new PartitionKey(conflictId));
+                iterator = containers.get(0).queryConflicts(query, options).byPage().toIterable().iterator();
+                readConflicts(iterator, testPojos, null);
+
+                //We will see some conflicts, as there is no sproc to resolve
+                assertThat(testPojos.size()).isEqualTo(containers.size() - 1);
+
+
+                //Testing queryConflicts(String query)
+                testPojos.clear();
+                iterator = containers.get(0).queryConflicts(query).byPage().toIterable().iterator();
+                readConflicts(iterator, testPojos, null);
+
+                //We will see some conflicts, as there is no sproc to resolve
+                assertThat(testPojos.size()).isEqualTo(containers.size() - 1);
+
+                for (String id : conflictIds) {
+                    CosmosConflictRequestOptions requestOptions =
+                        new CosmosConflictRequestOptions(new PartitionKey(conflictId));
+                    containers.get(0).getConflict(id).delete(requestOptions).block();
+                }
+                Thread.sleep(5000); // Wait for conflict item to replicate
+
+                iterator =
+                    containers.get(0).readAllConflicts(new CosmosQueryRequestOptions()).byPage().toIterable().iterator();
+                testPojos.clear();
+                readConflicts(iterator, testPojos, null);
+
+                //Making sure all conflicts are deleted
+                assertThat(testPojos.size()).isEqualTo(0);
+
+                CosmosItemResponse<ConflictTestPojo> itemResponse = containers.get(0).readItem(conflictId,
+                    new PartitionKey(conflictId), null, ConflictTestPojo.class).block();
+                //Verifying during conflict on create we will have one winner item saved
+                assertThat(itemResponse.getItem().getId()).isEqualTo(conflictId);
+            } finally {
+                database.getContainer(containerProperties.getId()).delete().block();
+            }
+        } else {
+            fail("Unable to find multi master account");
+        }
+    }
+
+    @AfterClass(groups = {"multi-master"}, timeOut = SHUTDOWN_TIMEOUT, alwaysRun = true)
+    public void afterClass() {
+        if (this.globalClient != null) {
+            this.globalClient.close();
```

Comment:
please use `TestSuiteBase:safeClose` that will do the null check.

Reply:
done